services/brig/src/Brig/Queue/Stomp.hs

```diff
@@ -132,6 +132,8 @@ enqueue b q m =
 -- configured properly, after failing on the same message several times the
 -- message will go into the Dead Letter Queue where it can be analyzed
 -- manually.
+--
+-- FUTUREWORK: This probably deserves a Polysemy action
```

Comment:
@isovector , I changed a TODO note here to a futurework note. We won't be introducing effects in this PR, hence I think a futurework note is more appropriate (we tend to avoid merging TODO notes).